influxdb3/src/commands/create.rs

```diff
@@ -231,6 +232,11 @@ pub struct TriggerConfig {
     disabled: bool,
     #[clap(long)]
     run_asynchronous: bool,
+    #[clap(long,
+        value_parser = ErrorBehavior::from_str,
```

Comment:
Will the options come out on the CLI help?

Reply:
Yeah, I improved this a bit just now.